### ISSUE TITLE
Ombi v4 stuck on coloured loading page, reverse proxy (nginx-proxy-manager)

### ISSUE THREAD

**Issue by Tragyn** [tidusjar/Ombi#3931]
**Describe the bug**
Clean install of Ombi, accessing it internally works, accessing it from subdomain (ombi.whiskeyjack.org) leaves me stuck on the "loading" page.

**Expected behavior**
Load ombi.

**Screenshots**
![12 12 20-15:07:46](https://user-images.githubusercontent.com/5403547/101995995-cf3e0100-3c8b-11eb-9b65-f6a158748daa.png)

**Logs (Logs directory where Ombi is located)**
Not sure what's going on here, but the .js and .css files aren't processing properly.

![12 12 20-15:08:14](https://user-images.githubusercontent.com/5403547/101996001-dd8c1d00-3c8b-11eb-847e-cb1e6a88b8c0.png)

**Desktop (please complete the following information):**
Docker on unRAID

**Ombi Version (please complete the following information):**
v4-preview

**Additional context**
Using nginx-proxy-manager
Settings:
![12 12 20-15:09:17](https://user-images.githubusercontent.com/5403547/101996019-03b1bd00-3c8c-11eb-860e-28ff1b5e9489.png)
![12 12 20-15:09:06](https://user-images.githubusercontent.com/5403547/101996020-044a5380-3c8c-11eb-8cf1-16bad58a2af7.png)

**Comment by JimneyHeffenbush** [comment]
I too am having this issue.
Accessing via the ipaddress - OK
Accessing via a hostname - Forever spinning folding cube

Running on QNAP via Docker.

**Comment by JoKerIsCraZy** [comment]
Same for me someones it just stuck in the Loading screen will i restart the Container on Docker

Running on an Synology via Docker

**Comment by Tragyn** [comment]
It's working now and I have no idea why.  I haven't changed anything..

**Comment by JimneyHeffenbush** [comment]
Mine is still down. Tried recreating, checking for a new build (linuxserver docker based on build v4.0.748-ls2 from 12/12/2020).Can access via IP address, but hangs at the loading screen if I use any type of hostname. Sent from Mail for Windows 10 From: TragynSent: Sunday, December 13, 2020 9:59 PMTo: tidusjar/OmbiCc: JimneyHeffenbush; CommentSubject: Re: [tidusjar/Ombi] Ombi v4 stuck on coloured loading page, reverse proxy (nginx-proxy-manager) (#3931) It's working now and I have no idea why. I haven't changed anything..—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by joe-eklund** [comment]
I am not able to reproduce this issue myself just yet since I had to implement the temporary fix I talked about [here](https://github.com/tidusjar/Ombi/issues/3483#issuecomment-728535921) in #3483. That fix is not great, but it had been working for me.

Just got another report today that it broke again on android. Works on desktop versions of chrome, firefox, etc., just android chrome that is broken. Seems like this is a duplicate of #3483 though, so maybe this should be close din favor of that one?
